src/fhda/tortoise.py

```diff
@@ -0,0 +1,256 @@
+import os
+import uuid
+import asyncio
+import copy
+from typing import List, Dict, Any, Optional, Callable
+import time
+import json
+from pydantic import BaseModel, Field
+from . import prompts
+
+from futurehouse_client import FutureHouseClient
+from futurehouse_client.models import TaskRequest, RuntimeConfig
+from futurehouse_client.models.app import AuthType
+
+
+class StepConfig(BaseModel):
+    """Configuration for a step in the pipeline"""
```

Comment:
[nit] can you end these docstrings in a period?

https://peps.python.org/pep-0257/#one-line-docstrings

> The docstring is a phrase ending in a period.